routemaster.rb

```diff
@@ -2,4 +2,30 @@ module Routemaster
   def self.now
     (Time.now.utc.to_f * 1e3).to_i
   end
+
+  DEFAULTS = {
+    redis_pool_size: 1
+  }.freeze
```

Comment:
well... try `const u_int32*` in C. You get a constant pointer to a mutable integer :)